drivers/QSPI.cpp

```diff
@@ -264,7 +264,7 @@ void QSPI::_build_qspi_command(int instruction, int address, int alt)
     if(address != -1) {
         _qspi_command.address.value = address; 
     } else {
-        _qspi_command.address.value = 0; 
+        _qspi_command.address.size = QSPI_CFG_ADDR_SIZE_NONE;
     }
```

Comment:
@Martin, the changes looks ok to me, although I wonder if your base version is not the latest. Can you please check on that? Overall, the changes looks ok to me.

Reply:
I am looking at this, I rebased it, locally seems all OK, and github does not show this one commit here (outdated), investigating